Libraries/LibWeb/TrustedTypes/TrustedTypePolicyFactory.h

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2025, Miguel Sacristán Izcue <miguel_tete17@hotmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <LibJS/Forward.h>
+#include <LibWeb/Bindings/PlatformObject.h>
+#include <LibWeb/Bindings/TrustedTypePolicyFactoryPrototype.h>
+
+namespace Web::TrustedTypes {
+
+class TrustedTypePolicyFactory final : public Bindings::PlatformObject {
+    WEB_PLATFORM_OBJECT(TrustedTypePolicyFactory, Bindings::PlatformObject);
+    GC_DECLARE_ALLOCATOR(TrustedTypePolicyFactory);
+
+public:
+    [[nodiscard]] static GC::Ref<TrustedTypePolicyFactory> create(JS::Realm&);
+
+    virtual ~TrustedTypePolicyFactory() override { }
+
+    Optional<String> get_attribute_type(String&, String&, Optional<String>, Optional<String>);
+
+private:
+    explicit TrustedTypePolicyFactory(JS::Realm&);
+    virtual void initialize(JS::Realm&) override;
+
+    Vector<String> created_policy_names = {};
```

Comment:
We use an `m_` prefix for member variables and we don't use `=` when initializing members. Typically for `Vector` members we would leave out the initializing braces entirely.

Reply:
Sure, even clang-tidy was complaining